src/LogicItLab/Salesforce/MapperBundle/Exception/FileHandlerNotInitializedException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace LogicItLab\Salesforce\MapperBundle\Exception;
+
+class FileHandlerNotInitializedException extends \Exception
+{
+
```

Comment:
Maybe it's nice to add a message here to say what went wrong. Makes debugging a lot easier in the future.